lib/prx_auth/rails/ext/controller.rb

```diff
@@ -116,7 +116,8 @@ def lookup_accounts(ids)
 
       def fetch_accounts(ids)
         ids_param = ids.map(&:to_s).join(',')
-        fetch("/api/v1/accounts?account_ids=#{ids_param}")['accounts']
+        resp = fetch("/api/v1/accounts?account_ids=#{ids_param}")
+        resp.try(:[], '_embedded').try(:[], 'prx:items') || []
```

Comment:
so my understanding is: because the HAL accounts are embedded into the API, that is why we are trying the `embedded` key followed by trying the `prx:items` key? what is the benefit of embedding vs. the other way?

Reply:
For _index_ endpoints in HAL APIs, you really always need to put the list-of-things under `_embedded`.  Since there aren't any top-level "attributes" as there would be when showing a resource.  Our convention is to put things under `prx:items` for index - but I've seen plenty of other things.

By "other way" ... you mean prior to this PR, where the data was nested under the `accounts` key?  There isn't much of a functional advantage - but this PR does bring the ID APIs into line with our others.  (CMS, Feeder, Augury, and Castle all have HAL APIs).